msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2023-05-09 09:37-0500\n"
"PO-Revision-Date: 2023-05-09 14:49\n"
"Last-Translator: \n"
"Language-Team: Spanish\n"
"Language: es_ES\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: duplicity\n"
"X-Crowdin-Project-ID: 539508\n"
"X-Crowdin-Language: es-ES\n"
"X-Crowdin-File: /main/po/duplicity.pot\n"
"X-Crowdin-File-ID: 6\n"

#: ../bin/duplicity:106
msgid "INT intercepted...exiting."
msgstr "INT interceptado... saliendo."

#: ../bin/duplicity:114
#, python-format
msgid "GPG error detail: %s"
msgstr "Detalle de error GPG: %s"

#: ../bin/duplicity:124
#, python-format
msgid "User error detail: %s"
msgstr "Detalle de error del usuario: %s"

#: ../bin/duplicity:134
#, python-format
msgid "Backend error detail: %s"
msgstr "Detalle de error del motor: %s"

#: ../bin/rdiffdir:64 ../duplicity/commandline.py:239
#, python-format
msgid "Error opening file %s"
msgstr "Error al abrir el archivo %s"

#: ../bin/rdiffdir:131
#, python-format
msgid "File %s already exists, will not overwrite."
msgstr "El archivo %s ya existe y no se sobrescribirá."

#: ../duplicity/tempdir.py:160
#, python-format
msgid "Using temporary directory %s"
msgstr "Utilizando directorio temporal %s"

#: ../duplicity/tempdir.py:204
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "Registrando archivo temporal (mktemp) %s"

#: ../duplicity/tempdir.py:226
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "Registrando archivo temporal (mkstemp) %s"

#: ../duplicity/tempdir.py:258
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Olvidando el archivo temporal %s"

#: ../duplicity/tempdir.py:261
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr "Intentando olvidar un archivo temporal desconocido %s. Esto es posiblemente un error."

#: ../duplicity/tempdir.py:280
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "Eliminar archivo temporal todavía recordado %s"

#: ../duplicity/tempdir.py:283
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "La limpieza del archivo temporal %s falló"

#: ../duplicity/tempdir.py:288
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr "La limpieza del directorio temporal %s falló. Esto posiblemente sea un error."

#: ../duplicity/backend.py:102
#, python-format
msgid "Import of %s %s"
msgstr "Importación de %s %s"

#: ../duplicity/backend.py:211
#, python-format
msgid "Could not initialize backend: %s"
msgstr "No se pudo inicializar el motor: %s"

#: ../duplicity/backend.py:403
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Volcado de pila del error anterior: %s"

#: ../duplicity/backend.py:419
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Desistiendo tras %s intentos. %s: %s"

#: ../duplicity/backend.py:423
#, python-format
msgid "Attempt of %s Nr. %s failed. %s: %s"
msgstr ""

#: ../duplicity/backend.py:517
#, python-format
msgid "Reading results of '%s'"
msgstr "Leyendo resultados de «%s»"

#: ../duplicity/backend.py:543
#, python-format
msgid "Writing %s"
msgstr "Escribiendo %s"

#: ../duplicity/backend.py:584
#, python-format
msgid "File %s not found locally after get from backend"
msgstr "archivo %s no encontrado localmente tras obtenerlo del servidor"

#: ../duplicity/manifest.py:99
#, python-format
msgid "Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr "Error fatal: el anfitrión de origen del respaldo ha cambiado.\n"
"Anfitrión actual: %s\n"
"Anfitrión anterior: %s"

#: ../duplicity/manifest.py:106
#, python-format
msgid "Fatal Error: Backup source directory has changed.\n"
"Current directory: %s\n"
"Previous directory: %s"
msgstr "Error fatal: el directorio de origen del respaldo ha cambiado.\n"
"Directorio actual: %s\n"
"Directorio anterior: %s"

#: ../duplicity/manifest.py:116
msgid "Aborting because you may have accidentally tried to backup two different data sets to the same remote location, or using the same archive directory.  If this is not a mistake, use the --allow-source-mismatch switch to avoid seeing this message"
msgstr "Abortar porque puede haber intentado dos conjuntos de datos distintos en la misma ubicación remota por error, o utilizado el mismo directorio de archivo. Sí no es un error, use --allow-source-mismatch para no ver este mensaje"

#: ../duplicity/manifest.py:218
#, python-format
msgid "Found manifest volume %s"
msgstr "Volumen manifest encontrado %s"

#: ../duplicity/manifest.py:225
#, python-format
msgid "Found %s volumes in manifest"
msgstr "Encontrados %s volúmenes en el manifiesto"

#: ../duplicity/manifest.py:243
#, python-format
msgid "Manifest file '%s' is corrupt: File count says %d, File list contains %d"
msgstr "El archivo de manifiesto «%s» está corrupto: El contador de archivos dice %d, La lista de archivos contiene %d"

#: ../duplicity/manifest.py:262
msgid "Manifests not equal because different volume numbers"
msgstr "Los manifiestos no son iguales debido a los diferentes números de volumen"

#: ../duplicity/manifest.py:267
msgid "Manifests not equal because volume lists differ"
msgstr "Los manifiestos no son iguales porque las listas de volúmenes difieren"

#: ../duplicity/manifest.py:272
msgid "Manifests not equal because hosts or directories differ"
msgstr "Los manifiestos no son iguales porque los anfitriones o directorios no coinciden"

#: ../duplicity/manifest.py:429
msgid "Warning, found extra Volume identifier"
msgstr "Aviso, se encontró un identificador de volumen extra"

#: ../duplicity/manifest.py:455
msgid "Other is not VolumeInfo"
msgstr "Otro no es VolumeInfo"

#: ../duplicity/manifest.py:458
msgid "Volume numbers don't match"
msgstr "Los números de volúmenes no coinciden"

#: ../duplicity/manifest.py:461
msgid "start_indicies don't match"
msgstr "start_indicies no coinciden"

#: ../duplicity/manifest.py:464
msgid "end_index don't match"
msgstr "end_index no coinciden"

#: ../duplicity/manifest.py:471
msgid "Hashes don't match"
msgstr "Hashes no coinciden"

#: ../duplicity/util.py:166 ../duplicity/dup_main.py:847
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr "IGNORED_ERROR: Aviso: ignorando error tal como se pidió: %s: %s"

#: ../duplicity/util.py:233
#, python-format
msgid "Releasing lockfile %s"
msgstr "Publicando fichero de candado %s"

#: ../duplicity/backends/pydrivebackend.py:181
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "PyDrive backend: se llamaron múltiples archivos «%s»."

#: ../duplicity/backends/multibackend.py:96
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "MultBackend: no se pudo analizar la cadena de consulta %s: %s "

#: ../duplicity/backends/multibackend.py:105
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr "MultBackend: cadena de consulta no válida %s: más de un valor para %s"

#: ../duplicity/backends/multibackend.py:110
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr "MultBackend: cadena de consulta no válida %s: parámetro desconocido %s"

#: ../duplicity/backends/multibackend.py:160
#: ../duplicity/backends/multibackend.py:165
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr "MultBackend: valor no permitido para %s: %s"

#: ../duplicity/backends/multibackend.py:176
#, python-format
msgid "MultiBackend: Url %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:180
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr "MultBackend: no se pudo cargar el archivo de configuración %s: %s "

#: ../duplicity/backends/multibackend.py:189
#, python-format
msgid "MultiBackend: use store %s"
msgstr "MultiBackend: usar almacén %s"

#: ../duplicity/backends/multibackend.py:194
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:207
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:248
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr "MultiBackend: _put: escribir en almacén #%s (%s)"

#: ../duplicity/backends/multibackend.py:261
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr "MultiBackend: falló al escribir en almacén #%s (%s), intenta #%s, Excepción: %s"

#: ../duplicity/backends/multibackend.py:268
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr "MultiBackend: falló al escribir %s. Abortando el proceso."

#: ../duplicity/backends/multibackend.py:275
#, python-format
msgid "MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend: falló al escribir %s. Se probaron todos los almacenes de copias y ninguno funcionó"

#: ../duplicity/backends/multibackend.py:294
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "MultiBackend: falló al obtener %s a %s de %s"

#: ../duplicity/backends/multibackend.py:297
#, python-format
msgid "MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend: falló al obtener %s. Se probaron todos los almacenes de copias y ninguno funcionó"

#: ../duplicity/backends/multibackend.py:307
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr ""

#: ../duplicity/backends/multibackend.py:310
#, python-format
msgid "Exception during list of %s: %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:317
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr "MultiBackend: lista combinada: %s"

#: ../duplicity/backends/multibackend.py:346
#: ../duplicity/backends/multibackend.py:375
#, python-format
msgid "MultiBackend: failed to delete %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend: falló al eliminar %s. Se probaron todos los almacenes de copias y ninguno funcionó"

#: ../duplicity/backends/giobackend.py:109
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Falló la conexión, compruebe su contraseña: %s"

#: ../duplicity/backends/onedrivebackend.py:92
#, python-format
msgid "Attempt of initialize_oauth2_session Nr. %s failed. %s: %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:67
msgid "Missing socket or ssl python modules."
msgstr "Faltan los módulos de Python «socket» o «ssl»."

#: ../duplicity/backends/webdavbackend.py:85
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr "Archivo de base de datos Cacert «%s» ilegible."

#: ../duplicity/backends/webdavbackend.py:106
msgid "Option '--ssl-cacert-path' is not supported with python 2.7.8 and below."
msgstr "Opción «--ssl-cacert-path» no es compatible con python 2.7.8 y anterior."

#: ../duplicity/backends/webdavbackend.py:110
#, python-format
msgid "For certificate verification with python 2.7.8 or earlier a cacert database\n"
"file is needed in one of these locations: %s\n"
"Hints:\n"
"Consult the man page, chapter 'SSL Certificate Verification'.\n"
"Consider using the options --ssl-cacert-file, --ssl-no-check-certificate ."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:168
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr "Usando servidor WebDAV %s puerto %s"

#: ../duplicity/backends/webdavbackend.py:170
#, python-format
msgid "Using WebDAV directory %s"
msgstr "Usando directorio de WebDAV %s"

#: ../duplicity/backends/webdavbackend.py:201
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr "Conexión para creación de WebDAV en «%s»"

#: ../duplicity/backends/webdavbackend.py:212
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr "Esquema de URI WebDAV desconocido: %s"

#: ../duplicity/backends/webdavbackend.py:231
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr "WebDAV %s %s petición con encabezados: %s "

#: ../duplicity/backends/webdavbackend.py:232
#: ../duplicity/backends/webdavbackend.py:255
#, python-format
msgid "WebDAV data length: %s "
msgstr "Longitud de datos de WebDAV: %s "

#: ../duplicity/backends/webdavbackend.py:235
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr "Estado de respuesta WebDAV %s con razón «%s»."

#: ../duplicity/backends/webdavbackend.py:241
#, python-format
msgid "WebDAV redirect to: %s "
msgstr "WebDAV redigir a: %s "

#: ../duplicity/backends/webdavbackend.py:243
msgid "WebDAV redirected 10 times. Giving up."
msgstr "WebDAV redirigido 10 veces. Abandonando."

#: ../duplicity/backends/webdavbackend.py:248
msgid "WebDAV missing location header in redirect response."
msgstr "A WebDAV le falta la cabecera de ubicación en la respuesta de redirección."

#: ../duplicity/backends/webdavbackend.py:253
msgid "WebDAV retry request with authentification headers."
msgstr "Petición de reintento de WebDAV con encabezados de autentificación."

#: ../duplicity/backends/webdavbackend.py:254
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr "WebDAV %s %s solicitud2 con encabezados: %s "

#: ../duplicity/backends/webdavbackend.py:258
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr "WebDAV respuesta2 estado %s con motivo «%s»."

#: ../duplicity/backends/webdavbackend.py:275
msgid "python-kerberos needed to use kerberos                           authorization, falling back to basic auth."
msgstr "python-kerberos necesitó usar la autorización kerberos, volviendo a la autorización básica."

#: ../duplicity/backends/webdavbackend.py:279
#, python-format
msgid "Kerberos authorization failed: %s.                          Falling back to basic auth."
msgstr "Falló autorización Kerberos: %s. Volviendo a autorización básica."

#: ../duplicity/backends/webdavbackend.py:377
#, python-format
msgid "Creating missing directory %s"
msgstr "Creando directorio que falta %s"

#: ../duplicity/backends/webdavbackend.py:381
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr "WebDAV MKCOL %s falló: %s %s"

#: ../duplicity/backends/webdavbackend.py:394
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr "Descodificación y traducción de rutas de WebDAV: %s → %s"

#: ../duplicity/backends/webdavbackend.py:439
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr "WebDAV GET Mal código de estado %s motivo %s."

#: ../duplicity/backends/webdavbackend.py:461
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr "WebDAV PUT Mal código de estado %s motivo %s."

#: ../duplicity/backends/webdavbackend.py:481
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr "WebDAV DEL Mal código de estado %s motivo %s."

#: ../duplicity/diffdir.py:115 ../duplicity/diffdir.py:418
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Error %s consiguiendo el delta para %s"

#: ../duplicity/diffdir.py:130
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Consiguiendo el delta de %s y %s"

#: ../duplicity/diffdir.py:181
#, python-format
msgid "A %s"
msgstr ""

#: ../duplicity/diffdir.py:188
#, python-format
msgid "M %s"
msgstr ""

#: ../duplicity/diffdir.py:210
#, python-format
msgid "Comparing %s and %s"
msgstr "Comparando %s y %s"

#: ../duplicity/diffdir.py:218
#, python-format
msgid "D %s"
msgstr ""

#: ../duplicity/commandline.py:78
#, python-format
msgid "Warning: Option %s is pending deprecation and will be removed in version 2.0.\n"
"Use of default filenames is strongly suggested."
msgstr ""

#: ../duplicity/commandline.py:84
#, python-format
msgid "Warning: Option %s is pending deprecation and will be removed in version 2.0.\n"
"--include-filelist and --exclude-filelist now accept globbing characters and should be used instead."
msgstr ""

#: ../duplicity/commandline.py:94
#, python-format
msgid "Warning: Option %s is pending deprecation and will be removed in version 2.0.\n"
"On many GNU/Linux systems, stdin is represented by /dev/stdin and\n"
"--include-filelist=/dev/stdin or --exclude-filelist=/dev/stdin could\n"
"be used as a substitute."
msgstr ""

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/commandline.py:261 ../duplicity/commandline.py:271
#: ../duplicity/commandline.py:292 ../duplicity/commandline.py:388
#: ../duplicity/commandline.py:410 ../duplicity/commandline.py:421
#: ../duplicity/commandline.py:669 ../duplicity/commandline.py:717
#: ../duplicity/commandline.py:953
msgid "path"
msgstr "ruta"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a hidden GnuPG key. Example:
#. --hidden-encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/commandline.py:287 ../duplicity/commandline.py:294
#: ../duplicity/commandline.py:416 ../duplicity/commandline.py:650
#: ../duplicity/commandline.py:926
msgid "gpg-key-id"
msgstr ""

#. Used in usage help to represent a pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <pattern>
#: ../duplicity/commandline.py:302 ../duplicity/commandline.py:444
#: ../duplicity/commandline.py:959
msgid "pattern"
msgstr ""

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/commandline.py:308 ../duplicity/commandline.py:317
#: ../duplicity/commandline.py:324 ../duplicity/commandline.py:392
#: ../duplicity/commandline.py:446 ../duplicity/commandline.py:453
#: ../duplicity/commandline.py:466 ../duplicity/commandline.py:922
msgid "filename"
msgstr "nombre de archivo"

#. Used in usage help to represent a regular expression (regexp).
#: ../duplicity/commandline.py:331 ../duplicity/commandline.py:457
msgid "regular_expression"
msgstr "expresión regular"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/commandline.py:335 ../duplicity/commandline.py:404
#: ../duplicity/commandline.py:544 ../duplicity/commandline.py:1008
msgid "time"
msgstr "hora"

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:412 ../duplicity/commandline.py:521
#: ../duplicity/commandline.py:547 ../duplicity/commandline.py:658
#: ../duplicity/commandline.py:941
msgid "options"
msgstr "opciones"

#: ../duplicity/commandline.py:430
#, python-format
msgid "Running in 'ignore errors' mode due to %s; please re-consider if this was not intended"
msgstr "Ejecutando en modo «ignore errors» debido a %s; reconsidere esta opción si no es su intención"

#. Used in usage help to represent an imap mailbox
#: ../duplicity/commandline.py:442
msgid "imap_mailbox"
msgstr ""

#: ../duplicity/commandline.py:460
msgid "file_descriptor"
msgstr ""

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/commandline.py:475 ../duplicity/commandline.py:497
#: ../duplicity/commandline.py:518 ../duplicity/commandline.py:524
#: ../duplicity/commandline.py:530 ../duplicity/commandline.py:583
#: ../duplicity/commandline.py:588 ../duplicity/commandline.py:592
#: ../duplicity/commandline.py:615 ../duplicity/commandline.py:621
#: ../duplicity/commandline.py:625 ../duplicity/commandline.py:699
#: ../duplicity/commandline.py:712 ../duplicity/commandline.py:725
#: ../duplicity/commandline.py:936
msgid "number"
msgstr "número"

#. Used in usage help (noun)
#: ../duplicity/commandline.py:478
msgid "backup name"
msgstr "nombre de respaldo"

#: ../duplicity/commandline.py:610
msgid "policy"
msgstr ""

#: ../duplicity/commandline.py:628
msgid "Hot|Cool|Archive"
msgstr ""

#. noun
#: ../duplicity/commandline.py:631 ../duplicity/commandline.py:634
#: ../duplicity/commandline.py:907
msgid "command"
msgstr "orden"

#: ../duplicity/commandline.py:637
msgid "pyrax|cloudfiles"
msgstr ""

#: ../duplicity/commandline.py:661
msgid "pem formatted bundle of certificate authorities"
msgstr "paquete formateado pem de entidades de certificación"

#: ../duplicity/commandline.py:662
msgid "path to a folder with certificate authority files"
msgstr "ruta a una carpeta con archivos de autoridad de certificados"

#: ../duplicity/commandline.py:666
msgid "extra headers for Webdav, like 'Cookie,name=value'"
msgstr ""

#. Used in usage help. Example:
#. --timeout <seconds>
#. Used in usage help. Example:
#. --backend-retry-delay <seconds>
#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/commandline.py:674 ../duplicity/commandline.py:730
#: ../duplicity/commandline.py:976
msgid "seconds"
msgstr "segundos"

#. abbreviation for "character" (noun)
#: ../duplicity/commandline.py:680 ../duplicity/commandline.py:904
msgid "char"
msgstr "carácter"

#: ../duplicity/commandline.py:736
#, python-format
msgid "Warning: Option %s is dangerous in that it allows corrupted manifests to\n"
"exist on the remote without being caught.  This could lead to problems\n"
"such as the inability to recover the backup in the future."
msgstr ""

#: ../duplicity/commandline.py:870
#, python-format
msgid "Using archive dir: %s"
msgstr "Usando directorio de archivador: %s"

#: ../duplicity/commandline.py:871
#, python-format
msgid "Using backup name: %s"
msgstr "Usando nombre de copia de seguridad: %s"

#: ../duplicity/commandline.py:878
#, python-format
msgid "Command line error: %s"
msgstr "Error de línea de órdenes: %s"

#: ../duplicity/commandline.py:879
msgid "Enter 'duplicity --help' for help screen."
msgstr "Introduzca «duplicity --help» para tener ayuda en pantalla."

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/commandline.py:892
msgid "absolute_path"
msgstr "ruta_absoluta"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/commandline.py:896
msgid "alias"
msgstr ""

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:901
msgid "bucket_name"
msgstr "nombre_cubo"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/commandline.py:912
msgid "container_name"
msgstr "nombre_contenedor"

#. noun
#: ../duplicity/commandline.py:915
msgid "count"
msgstr "recuento"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/commandline.py:918
msgid "directory"
msgstr "directorio"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/commandline.py:931
msgid "module"
msgstr "módulo"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:945
msgid "other.host"
msgstr "otro.host"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:949
msgid "password"
msgstr "contraseña"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:963
msgid "port"
msgstr "puerto"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:968
msgid "prefix"
msgstr "prefijo"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/commandline.py:972
msgid "relative_path"
msgstr "ruta_relativa"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/commandline.py:981
msgid "some_dir"
msgstr "algun_dir"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:987
msgid "source_dir"
msgstr "dir_origen"

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:992
msgid "source_url"
msgstr "url_origen"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:998
msgid "target_dir"
msgstr "dir_objetivo"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:1003
msgid "target_url"
msgstr "url_objetivo"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:1013
msgid "user"
msgstr "usuario"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/commandline.py:1016
msgid "account_id"
msgstr "id_cuenta"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/commandline.py:1020
msgid "application_key"
msgstr "clave_aplicación"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/commandline.py:1024 ../duplicity/dup_collections.py:521
msgid "remote"
msgstr "remoto"

#. Header in usage help
#: ../duplicity/commandline.py:1043
msgid "Backends and their URL formats:"
msgstr "Motores y sus formatos de URL:"

#. Header in usage help
#: ../duplicity/commandline.py:1081
msgid "Commands:"
msgstr "Órdenes:"

#: ../duplicity/commandline.py:1108
#, python-format
msgid "Specified archive directory '%s' does not exist, or is not a directory"
msgstr "El directorio del archivador «%s» no existe, o no es un directorio"

#: ../duplicity/commandline.py:1117
#, python-format
msgid "Sign key should be an 8, 16 alt. 40 character hex string, like 'AA0E73D2'.\n"
"Received '%s' instead."
msgstr "La clave de firma debería ser una cadena de 8, 16 o 40 caracteres hexadecimales , como «AA0E732D2».\n"
"Se ha recibido «%s» en su lugar."

#: ../duplicity/commandline.py:1177
#, python-format
msgid "Restore destination directory %s already exists.\n"
"Will not overwrite."
msgstr "Restaurar directorio de destino %s ya existe.\n"
"No sobrescribir."

#: ../duplicity/commandline.py:1182
#, python-format
msgid "Verify directory %s does not exist"
msgstr "El directorio verificado %s no existe"

#: ../duplicity/commandline.py:1188
#, python-format
msgid "Backup source directory %s does not exist."
msgstr "El directorio de origen de respaldo %s no existe."

#: ../duplicity/commandline.py:1219
#, python-format
msgid "Command line warning: %s"
msgstr "Aviso de línea de órdenes: %s"

#: ../duplicity/commandline.py:1219
msgid "Selection options --exclude/--include\n"
"currently work only when backing up,not restoring."
msgstr "Las opciones de selección --exclude/--include\n"
"actualmente funcionan solo cuando se respalda no al restaurar."

#: ../duplicity/commandline.py:1255
#, python-format
msgid "GPG binary is %s, version %s"
msgstr "El binario de GPG es %s, versión %s"

#: ../duplicity/commandline.py:1282
#, python-format
msgid "Bad URL '%s'.\n"
"Examples of URL strings are \"scp://user@host.net:1234/path\" and\n"
"\"file:///usr/local\".  See the man page for more information."
msgstr "URL erróneo «%s».\n"
"Ejemplos de cadenas de URL son «scp://user@host.net:1234/path» \n"
"y «file:///usr/local». Ver el manual para más información."

#: ../duplicity/commandline.py:1312
msgid "Main action: "
msgstr "Acción principal: "

#: ../duplicity/librsync.py:196
msgid "basis_file must be a (true) file or an object whose file attribute is the underlying true file object"
msgstr "basis_file debe ser un archivo (verdadero) o un objeto cuyos atributos de archivo son el objeto del archivo verdadero subyacente"

#: ../duplicity/dup_time.py:63
#, python-format
msgid "Bad interval string \"%s\"\n\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr "Cadena de intervalo errónea «%s»\n\n"
"Los intervalos son del tipo 2Y (2 años) o 2h30m (2.5 horas). \n"
"Los caracteres especiales permitidos son s, m, h, D, W, M e Y.\n"
"Ver el manual para más información."

#: ../duplicity/dup_time.py:69
#, python-format
msgid "Bad time string \"%s\"\n\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr "Cadena de fecha incorrecta «%s»\n\n"
"Las cadenas de fecha aceptable son intervalos (tipo «3D64s»), cadenas w3-datetime\n"
", tipo «2002-04-26T04:22:01-07:00» (cadenas tipo\n"
"«2002-04-26T04:22:01» son aceptables - duplicity usará la zona\n"
"horaria actual), o fecha normales tipo 2/4/1997 o 2001-04-23\n"
"(son aceptables distintas combinaciones, pero el mes debe preceder\n"
" al día siempre)."

#: ../duplicity/dup_main.py:120
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Reusar configurar PASSPHRASE como SIGN_PASSPHRASE"

#: ../duplicity/dup_main.py:127
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Reusar configurar SIGN_PASSPHRASE como PASSPHRASE"

#: ../duplicity/dup_main.py:168
msgid "PASSPHRASE variable not set, asking user."
msgstr "Variable PASSPHRASE no establecida, preguntar al usuario."

#: ../duplicity/dup_main.py:183
msgid "GnuPG passphrase for signing key:"
msgstr "Frase de paso GnuPG para clave de firmada:"

#: ../duplicity/dup_main.py:188
msgid "GnuPG passphrase for decryption:"
msgstr ""

#: ../duplicity/dup_main.py:193
msgid "Retype passphrase for signing key to confirm: "
msgstr "Vuelva a teclear la frase de paso de la clave firmada para confirmar: "

#: ../duplicity/dup_main.py:195
msgid "Retype passphrase for decryption to confirm: "
msgstr ""

#: ../duplicity/dup_main.py:198
msgid "First and second passphrases do not match!  Please try again."
msgstr "¡La primera y segunda contraseña no coinciden! Por favor inténtelo de nuevo."

#: ../duplicity/dup_main.py:205
msgid "Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr "¡No puede usar una contraseña en blanco con cifrado asimétrico! Por favor inténtelo de nuevo."

#: ../duplicity/dup_main.py:263
#, python-format
msgid "File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr "El archivo %s está competo en el conjunto de respaldo.\n"
"Continuar la restauración en el archivo %s."

#: ../duplicity/dup_main.py:272
#, python-format
msgid "File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr "El archivo %s  falta en el conjunto de respaldo.\n"
"Reinicio continuo en el archivo %s."

#: ../duplicity/dup_main.py:326
#, python-format
msgid "%s Remote filesize %d for %s does not match local size %d, retrying."
msgstr ""

#: ../duplicity/dup_main.py:331
#, python-format
msgid "File %s was corrupted during upload."
msgstr "El archivo %s se ha dañado en la carga."

#: ../duplicity/dup_main.py:371
msgid "Restarting backup, but current encryption settings do not match original settings"
msgstr "Reiniciando copia de respaldo, pero la configuración actual de cifrado no coincide con la configuración original"

#: ../duplicity/dup_main.py:394
msgid "Skipping encryption validation due to glacier/deep storage"
msgstr ""

#: ../duplicity/dup_main.py:397
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "Reiniciando tras volumen %s, archivo %s, bloque %s"

#: ../duplicity/dup_main.py:475
#, python-format
msgid "Processed volume %d"
msgstr "Se procesó el volumen %d"

#: ../duplicity/dup_main.py:625
msgid "Fatal Error: Unable to start incremental backup.  Old signatures not found and incremental specified"
msgstr "Error Fatal: No se puede iniciar respaldo incremental. Firmas de edad y que no se encuentran especificados incremental"

#: ../duplicity/dup_main.py:629
msgid "No signatures found, switching to full backup."
msgstr "Firmas no encontradas, cambiando a un backup completo."

#: ../duplicity/dup_main.py:643
msgid "Backup Statistics"
msgstr "Estadísticas de respaldo"

#: ../duplicity/dup_main.py:751
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "No se encontró %s en el archivador; no se restauró ningún archivo."

#: ../duplicity/dup_main.py:755
msgid "No files found in archive - nothing restored."
msgstr "No se encontraron archivos en el archivador - nada que restaurar."

#: ../duplicity/dup_main.py:792
#, python-format
msgid "Processed volume %d of %d"
msgstr "Volumen procesado %d de %d"

#: ../duplicity/dup_main.py:837
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "datos no válidos - %s hash no coincide con archivo:"

#: ../duplicity/dup_main.py:840
#, python-format
msgid "Calculated hash: %s"
msgstr "hash calculado: %s"

#: ../duplicity/dup_main.py:841
#, python-format
msgid "Manifest hash: %s"
msgstr "Manifestar hass: %s"

#: ../duplicity/dup_main.py:892
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "El volumen fue firmado por %s clave, no %s"

#: ../duplicity/dup_main.py:924
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Verificación completa: %s, %s."

#: ../duplicity/dup_main.py:925
#, python-format
msgid "%d file(s) compared"
msgstr ""

#: ../duplicity/dup_main.py:926
#, python-format
msgid "%d difference(s) found"
msgstr ""

#: ../duplicity/dup_main.py:944
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Ningún archivo raro encontrado, nada borrado al acabar."

#: ../duplicity/dup_main.py:949
msgid "Deleting these file(s) from backend:"
msgstr ""

#: ../duplicity/dup_main.py:958
msgid "Found the following file(s) to delete:"
msgstr ""

#: ../duplicity/dup_main.py:959
msgid "Run duplicity again with the --force option to actually delete."
msgstr "Ejecutar duplicity de nuevo con la opción --foirce para eliminar realmente."

#: ../duplicity/dup_main.py:1002
msgid "There are backup set(s) at time(s):"
msgstr "Hay conjuntos de respaldo con hora:"

#: ../duplicity/dup_main.py:1004
msgid "Which can't be deleted because newer sets depend on them."
msgstr "Que no se puede eliminar porque los nuevos conjuntos dependen de ellos."

#: ../duplicity/dup_main.py:1008
msgid "Current active backup chain is older than specified time.  However, it will not be deleted.  To remove all your backups, manually purge the repository."
msgstr "La cadena del respaldo activo actual es anterior a la hora especificada, por lo que no se eliminará. Para eliminar todos sus respaldos, purgue manualmente el repositiorio."

#: ../duplicity/dup_main.py:1021
msgid "No old backup sets found, nothing deleted."
msgstr "No se encontraron respaldos. nada que eliminar."

#: ../duplicity/dup_main.py:1024
msgid "Deleting backup chain(s) at time:"
msgstr ""

#: ../duplicity/dup_main.py:1034
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr "Eliminando cualquier cadena de firmado incremental con origen en %s"

#: ../duplicity/dup_main.py:1036
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr "Eliminando cualquier copia de seguridad incremental con origen en %s"

#: ../duplicity/dup_main.py:1039
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "Borrando cadenas de firmas completas %s"

#: ../duplicity/dup_main.py:1041
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Borrando cadena de copias de seguridad completas %s"

#: ../duplicity/dup_main.py:1047
msgid "Found old backup chain(s) at the following time:"
msgstr ""

#: ../duplicity/dup_main.py:1049
msgid "Rerun command with --force option to actually delete."
msgstr "Volviendo a ejecutar orden con la opción --force para eliminar realmente."

#: ../duplicity/dup_main.py:1072
msgid "No old backup sets found."
msgstr ""

#: ../duplicity/dup_main.py:1087
#, python-format
msgid "Signature %s already replicated"
msgstr ""

#: ../duplicity/dup_main.py:1094 ../duplicity/dup_main.py:1133
#, python-format
msgid "Replicating %s."
msgstr ""

#: ../duplicity/dup_main.py:1121
#, python-format
msgid "Backupset %s already replicated"
msgstr ""

#: ../duplicity/dup_main.py:1269
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "Eliminación local %s (No es una autoridad en el procesamiento)"

#: ../duplicity/dup_main.py:1274
#, python-format
msgid "Unable to delete %s: %s"
msgstr "No se puede eliminar %s: %s"

#: ../duplicity/dup_main.py:1308 ../duplicity/dup_temp.py:271
#, python-format
msgid "Failed to read %s: %s"
msgstr "No se pudo leer %s: %s"

#: ../duplicity/dup_main.py:1321
#, python-format
msgid "Copying %s to local cache."
msgstr "Copiando %s a la caché local."

#: ../duplicity/dup_main.py:1369
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr "Los metadatos en local y remoto están sincronizados, no es necesario sincronizar."

#: ../duplicity/dup_main.py:1374
msgid "Synchronizing remote metadata to local cache..."
msgstr "Sincronizando metadatos remotos con la caché local..."

#: ../duplicity/dup_main.py:1387
msgid "Sync would copy the following from remote to local:"
msgstr "Sync copiaría lo siguiente de remoto a local:"

#: ../duplicity/dup_main.py:1390
msgid "Sync would remove the following spurious local files:"
msgstr "Sync eliminaría los siguientes archivos locales:"

#: ../duplicity/dup_main.py:1434
msgid "Unable to get free space on temp."
msgstr "Imposible conseguir espacio libre en temp."

#: ../duplicity/dup_main.py:1442
#, python-format
msgid "Temp space has %d available, backup needs approx %d."
msgstr "Espacio temporal tiene %d disponibles, copias de seguridad necesitan aproximadamente %d."

#: ../duplicity/dup_main.py:1445
#, python-format
msgid "Temp has %d available, backup will use approx %d."
msgstr "Temp tiene %d disponible, el respaldo usará aproximadamente %d."

#: ../duplicity/dup_main.py:1453
msgid "Unable to get max open files."
msgstr "No se puede obtener archivos de máxima apertura."

#: ../duplicity/dup_main.py:1457
#, python-format
msgid "Max open files of %s is too low, should be >= 1024.\n"
"Use 'ulimit -n 1024' or higher to correct.\n"
msgstr "El máximo de archivos abiertos %s es demasiado pequeño, debería ser >= 1.024.\n"
"Utilice «ulimit -n 1024» o más alto para corregirlo.\n"

#: ../duplicity/dup_main.py:1509
msgid "RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr "REINICIAR: El primer volumen no se han cargado antes de la terminación.\n"
"         Reiniciar es imposible... iniciando respaldo desde el principio."

#: ../duplicity/dup_main.py:1515
#, python-format
msgid "RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr "REINICIAR: Volúmenes %d a %d fallaron al cargar antes de terminar.\n"
"         Reiniciar copia de respaldo en el volumen %d."

#: ../duplicity/dup_main.py:1522
#, python-format
msgid "RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last partial\n"
"         backup then restart the backup from the beginning."
msgstr "REINICIAR: Estado de copia de respaldo no posible: manifesto tiene %d vols, remoto tiene %d vols.\n"
"         Reiniciar no es posible .. duplicity limpiará la última copia de respaldo parcial\n"
"         y reiniciará la copia desde el principio."

#: ../duplicity/dup_main.py:1543
msgid "\n"
"PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"the environment and rerun the backup.\n\n"
"See https://bugs.launchpad.net/duplicity/+bug/931175\n"
msgstr "\n"
"PYTHONOPTIMIZE en el entorno causa que duplicity falle al\n"
"reconocer sus propias copias de seguridad. Elimine PYTHONOPTIMIZE\n"
"del entorno y vuelva a ejecutar la copia de seguridad.\n\n"
"Consulte https://bugs.launchpad.net/duplicity/+bug/931175\n"

#: ../duplicity/dup_main.py:1566
#, python-format
msgid "Acquiring lockfile %s"
msgstr "Adquiriendo candado %s"

#: ../duplicity/dup_main.py:1629
#, python-format
msgid "Last %s backup left a partial set, restarting."
msgstr "La última copia de seguridad %s dejó una configuración parcial, reiniciar."

#: ../duplicity/dup_main.py:1633
#, python-format
msgid "Cleaning up previous partial %s backup set, restarting."
msgstr "Limpiar la configuración parcial de la copia de seguridad %s previa, reiniciar."

#: ../duplicity/dup_main.py:1645
msgid "Last full backup date:"
msgstr "Fecha del último respaldo completo:"

#: ../duplicity/dup_main.py:1647
msgid "Last full backup date: none"
msgstr "Fecha del último respaldo completo: ninguna"

#: ../duplicity/dup_main.py:1650
msgid "Last full backup is too old, forcing full backup"
msgstr "El último respaldo completo es demasiado viejo, lo que obligó a una copia de seguridad completa"

#: ../duplicity/dup_main.py:1699
msgid "When using symmetric encryption, the signing passphrase must equal the encryption passphrase."
msgstr "Al usar el cifrado simétrico, la contraseña de firmado debe coincidir con la de cifrado."

#: ../duplicity/asyncscheduler.py:70
#, python-format
msgid "instantiating at concurrency %d"
msgstr "crear una instancia en la concurrencia %d"

#: ../duplicity/asyncscheduler.py:97
msgid "inserting barrier"
msgstr "insertando barrera"

#: ../duplicity/asyncscheduler.py:146
msgid "running task synchronously (asynchronicity disabled)"
msgstr "ejecución de tareas de forma sincrónica (asincronía desactivada)"

#: ../duplicity/asyncscheduler.py:152
msgid "scheduling task for asynchronous execution"
msgstr "programación de tareas para la ejecución asíncrona"

#: ../duplicity/asyncscheduler.py:181
msgid "task completed successfully"
msgstr "tarea completada satisfactoriamente"

#: ../duplicity/asyncscheduler.py:192
msgid "a previously scheduled task has failed; propagating the result immediately"
msgstr "una tarea planificada previamente falló; propagando el resultado inmediatamente"

#: ../duplicity/asyncscheduler.py:215 ../duplicity/asyncscheduler.py:236
#, python-format
msgid "active workers = %d"
msgstr "trabajadores activos = %d"

#: ../duplicity/asyncscheduler.py:256
#, python-format
msgid "task execution done (success: %s)"
msgstr "ejecución de la tarea realizada (éxito: %s)"

#: ../duplicity/gpginterface.py:241
msgid "Threading not available -- zombie processes may appear"
msgstr "Hilo no disponible -- pueden aparecer procesos zombis"

#: ../duplicity/gpginterface.py:710
#, python-format
msgid "GPG process %d terminated before wait()"
msgstr "El proceso GPG %d terminó antes que el procedimiento wait()"

#: ../duplicity/robust.py:64
#, python-format
msgid "Error listing directory %s"
msgstr "Error al escuchar el directorio %s"

#: ../duplicity/dup_collections.py:186 ../duplicity/dup_collections.py:200
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "BackupSet.delete: falta %s"

#: ../duplicity/dup_collections.py:225
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "Error fatal: no se encontraron manifiestos para el respaldo más reciente"

#: ../duplicity/dup_collections.py:235
msgid "Fatal Error: Remote manifest does not match local one.  Either the remote backup set or the local archive directory has been corrupted."
msgstr "Error fatal: el manifiesto remoto no coincide con el local. Ni la configuración de copia de seguridad remota o el directorio del archivo local está dañado."

#: ../duplicity/dup_collections.py:240 ../duplicity/dup_collections.py:270
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr "Error al procesar el manifiesto remoto (%s): %s"

#: ../duplicity/dup_collections.py:247
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr "Error fatal: ni el manifiesto remoto ni el local son legibles"

#: ../duplicity/dup_collections.py:258
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr "Procesando manifiesto local %s (%s)"

#: ../duplicity/dup_collections.py:272
#, python-format
msgid "Processing remote manifest %s (%s)"
msgstr "Procesando manifiesto remoto %s (%s)"

#: ../duplicity/dup_collections.py:368
msgid "Preferring Backupset over previous one!"
msgstr "Preferir la configuración de copia de seguridad previa"

#: ../duplicity/dup_collections.py:371
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr "Ignorar la configuración de copia de seguridad incremental (start_time: %s; necesaria: %s)"

#: ../duplicity/dup_collections.py:376
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr "Añadida configuración de copia de seguridad incremental (start_time: %s / end_time: %s)"

#: ../duplicity/dup_collections.py:446
msgid "Chain start time: "
msgstr "Hora de inicio de la cadena: "

#: ../duplicity/dup_collections.py:447
msgid "Chain end time: "
msgstr "Hora de terminación de la cadena: "

#: ../duplicity/dup_collections.py:448
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Número de conjuntos de respaldo contenidos: %d"

#: ../duplicity/dup_collections.py:450
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Número total de volúmenes contenidos: %d"

#: ../duplicity/dup_collections.py:452 ../duplicity/dup_collections.py:1228
msgid "Type of backup set:"
msgstr "Tipo de conjunto de respaldo"

#: ../duplicity/dup_collections.py:452 ../duplicity/dup_collections.py:1228
msgid "Time:"
msgstr "Hora:"

#: ../duplicity/dup_collections.py:452
msgid "Num volumes:"
msgstr "Número de volúmenes:"

#: ../duplicity/dup_collections.py:456 ../duplicity/dup_collections.py:1234
msgid "Full"
msgstr "Completo"

#: ../duplicity/dup_collections.py:459 ../duplicity/dup_collections.py:1236
msgid "Incremental"
msgstr ""

#: ../duplicity/dup_collections.py:519
msgid "local"
msgstr ""

#: ../duplicity/dup_collections.py:676
msgid "Collection Status"
msgstr "Estado de la colección"

#: ../duplicity/dup_collections.py:678
#, python-format
msgid "Connecting with backend: %s"
msgstr "Conectar con el motor: %s"

#: ../duplicity/dup_collections.py:680
#, python-format
msgid "Archive dir: %s"
msgstr "Directorio de archivador: %s"

#: ../duplicity/dup_collections.py:682
#, python-format
msgid "Found %d secondary backup chain(s)."
msgstr ""

#: ../duplicity/dup_collections.py:685
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Cadena secundaria %d de %d:"

#: ../duplicity/dup_collections.py:691
msgid "Found primary backup chain with matching signature chain:"
msgstr "Se encontró cadena de copia de seguridad primaria con cadena de firma coincidente:"

#: ../duplicity/dup_collections.py:695
msgid "No backup chains with active signatures found"
msgstr "No encontró cadenas con firmas activas"

#: ../duplicity/dup_collections.py:698
#, python-format
msgid "Also found %d backup set(s) not part of any chain,"
msgstr ""

#: ../duplicity/dup_collections.py:700
#, python-format
msgid "and %d incomplete backup set(s)."
msgstr ""

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:703
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr "Esto se puede eliminar ejecutando duplicity con la orden «cleanup»"

#: ../duplicity/dup_collections.py:706
msgid "No orphaned or incomplete backup sets found."
msgstr "No se han encontrado respaldos huérfanos o incompletos."

#: ../duplicity/dup_collections.py:722
#, python-format
msgid "%d file(s) exists on backend"
msgstr ""

#: ../duplicity/dup_collections.py:730
#, python-format
msgid "%d file(s) exists in cache"
msgstr ""

#: ../duplicity/dup_collections.py:781
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr "Aviso, descartando el último conjunto de respaldo, debido a la falta archivo de firma."

#: ../duplicity/dup_collections.py:804
msgid "Warning, found the following local orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:809
msgid "Warning, found the following remote orphaned signature file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:814
msgid "Warning, found signatures but no corresponding backup files"
msgstr "Aviso, se encontraron firmas pero no coinciden con los archivos de copia de seguridad"

#: ../duplicity/dup_collections.py:818
msgid "Warning, found incomplete backup sets, probably left from aborted session"
msgstr "Aviso, encontró conjuntos de copia de seguridad incompletos, probablemente de la sesión abortada"

#: ../duplicity/dup_collections.py:822
msgid "Warning, found the following orphaned backup file(s):"
msgstr ""

#: ../duplicity/dup_collections.py:835
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Extrayendo cadenas de respaldo de la lista de archivos: %s"

#: ../duplicity/dup_collections.py:847
#, python-format
msgid "File %s is part of known set"
msgstr "El archivo %s es parte del conjunto conocido"

#: ../duplicity/dup_collections.py:850
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "El archivo %s no es parte de un conjunto conocido; creando un conjunto nuevo"

#: ../duplicity/dup_collections.py:855
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Ignorando archivo (rechazado por el conjunto) «%s»"

#: ../duplicity/dup_collections.py:871
#, python-format
msgid "Found backup chain %s"
msgstr "Se encontró una cadena de respaldo %s"

#: ../duplicity/dup_collections.py:876
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "conjunto %s añadido a la cadena %s preexistente"

#: ../duplicity/dup_collections.py:880
#, python-format
msgid "Found orphaned set %s"
msgstr "Se encontró un conjunto huérfano %s"

#: ../duplicity/dup_collections.py:1030
#, python-format
msgid "No signature chain for the requested time. Using oldest available chain, starting at time %s."
msgstr "No hay cadena de firmas disponibles para la hora solicitada. Se está usando la cadena más antigua disponible, que comienza en %s."

#: ../duplicity/dup_collections.py:1226
#, python-format
msgid "File: %s"
msgstr "Archivo: %s"

#: ../duplicity/dup_collections.py:1227
#, python-format
msgid "Total number of backup: %d"
msgstr "Número total de copias de seguridad: %d"

#: ../duplicity/dup_collections.py:1228 ../duplicity/dup_collections.py:1254
msgid "Type of file change:"
msgstr "Tipo de cambio de archivo:"

#: ../duplicity/dup_collections.py:1252
#, python-format
msgid " Backup set time: %s"
msgstr ""

#: ../duplicity/dup_collections.py:1253
#, python-format
msgid "Total number of changes: %d"
msgstr ""

#: ../duplicity/dup_collections.py:1254
msgid "File:"
msgstr ""

#: ../duplicity/patchdir.py:83 ../duplicity/patchdir.py:88
#, python-format
msgid "Patching %s"
msgstr "Parcheando %s"

#: ../duplicity/patchdir.py:558
#, python-format
msgid "Error '%s' patching %s"
msgstr "Error «%s» parcheando %s"

#: ../duplicity/patchdir.py:633
#, python-format
msgid "Writing %s of type %s"
msgstr "Escribiendo %s del tipo %s"

#: ../duplicity/selection.py:117
#, python-format
msgid "Skipping socket %s"
msgstr "Omitiendo el zócalo %s"

#: ../duplicity/selection.py:121
#, python-format
msgid "Error initializing file %s"
msgstr "Error al inicializar el archivo %s"

#: ../duplicity/selection.py:125 ../duplicity/selection.py:164
#: ../duplicity/selection.py:583
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Error al acceder a un archivo posiblemente bloqueado %s"

#: ../duplicity/selection.py:179
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Aviso: %s base no existe, continuando"

#: ../duplicity/selection.py:182 ../duplicity/selection.py:203
#: ../duplicity/selection.py:206
#, python-format
msgid "Selecting %s"
msgstr "Seleccionando %s"

#: ../duplicity/selection.py:259
#, python-format
msgid "The last file selection option is the filter option %s, which will have no\n"
"effect as there are no subsequent file selection options. Exiting because\n"
"this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:268
#, python-format
msgid "Only these filter mode options were specified:\n"
"    %s\n"
"Case sensitive globbing is the default behaviour and so this has no effect.\n"
"Exiting because this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:342
#, python-format
msgid "Fatal Error: The file specification\n"
"    %s\n"
"cannot match any files in the base directory\n"
"    %s\n"
"Useful file specifications begin with the base directory or some\n"
"pattern (such as '**') which matches the base directory."
msgstr "Error fatal: El archivo especificado\n"
"    %s\n"
"no coincide con ningún archivo del directorio principal\n"
"    %s\n"
"Las especificaciones de archivos útiles empiezan con el directorio\n"
"principal o pautas (tipo «**«) que coinciden con el directorio principal."

#: ../duplicity/selection.py:351
#, python-format
msgid "Fatal Error while processing expression\n"
"%s"
msgstr "Error fatal al procesar la expresión\n"
"%s"

#: ../duplicity/selection.py:368
#, python-format
msgid "Reading files to backup from list %s"
msgstr ""

#: ../duplicity/selection.py:390
#, python-format
msgid "Files-from list contains the absolute path:\n"
"    %s\n"
"All paths specified in a files-from list must be given relative to the backup\n"
"source path."
msgstr ""

#: ../duplicity/selection.py:399
msgid "Files-from list specified which contains no files, the backup will be empty as\n"
"a result. Exiting as this probably isn't what you meant,"
msgstr ""

#: ../duplicity/selection.py:409
#, python-format
msgid "Last selection expression:\n"
"    %s\n"
"only specifies that files be included.  Because the default is to\n"
"include all files, the expression is redundant.  Exiting because this\n"
"probably isn't what you meant."
msgstr "La última expresión elegida:\n"
"    %s\n"
"solo especifica qué archivos están incluidos. Porque la expresión de serie\n"
"incluye todos los archivos, es redundante. Salir porque probablemente\n"
"no es la opción elegida."

#: ../duplicity/selection.py:468
#, python-format
msgid "Reading %s filelist %s"
msgstr ""

#: ../duplicity/selection.py:509
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Error al compilar la expresión regular %s"

#: ../duplicity/lazy.py:355
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Aviso: índice antiguo %s >= nuevo índice %s"

#: ../duplicity/lazy.py:430
#, python-format
msgid "Error '%s' processing %s"
msgstr "Error «%s» procesando %s"

#: ../duplicity/lazy.py:440
#, python-format
msgid "Skipping %s because of previous error"
msgstr "Omitiendo %s debido al error anterior"

#: ../duplicity/path.py:115
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr "Aviso: %s devnums inválidos (0x%X), tratando como (0, 0)."

#: ../duplicity/path.py:245 ../duplicity/path.py:306
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Aviso: %s tiene mtime negativo, tratando como 0."

#: ../duplicity/path.py:370
msgid "Difference found:"
msgstr "Diferencia encontrada:"

#: ../duplicity/path.py:379
#, python-format
msgid "New file %s"
msgstr "Archiv nuevo %s"

#: ../duplicity/path.py:382
#, python-format
msgid "File %s is missing"
msgstr "Falta el archivo %s"

#: ../duplicity/path.py:385
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "El archivo %%s tiene el tipo %s, se esperaba %s"

#: ../duplicity/path.py:391 ../duplicity/path.py:417
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "El archivo %%s tiene permisos %s, se esperaba %s"

#: ../duplicity/path.py:396
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "El archivo %%s tiene mtime %s, se esperaba %s"

#: ../duplicity/path.py:404
#, python-format
msgid "Data for file %s is different"
msgstr "Los datos para el archivo %s son diferentes"

#: ../duplicity/path.py:412
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "El vínculo simbólico %%s apunta a %s, se esperaba %s"

#: ../duplicity/path.py:421
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "El archivo de dispositivo %%s tiene números %s, se esperaba %s"

#: ../duplicity/path.py:613
#, python-format
msgid "Making directory %s"
msgstr "Haciendo directorio %s"

#: ../duplicity/path.py:623
#, python-format
msgid "Deleting %s"
msgstr "Borrando %s"

#: ../duplicity/path.py:632
#, python-format
msgid "Touching %s"
msgstr "Tocando %s"

#: ../duplicity/path.py:639
#, python-format
msgid "Deleting tree %s"
msgstr "Eliminando árbol %s"

